cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/beans/ServiceBean.java

```diff
@@ -57,7 +57,13 @@ public void setLastOperation(String lastOperation) {
         this.lastOperation = lastOperation;
     }
 
+    /**
+     * @return the name if it is set, the service otherwise
```

Comment:
Not sure if this really cuts it. What exactly is the "service" here? Is it some sort of auto-generated identifier?